emrun.py

```diff
@@ -491,14 +512,14 @@ def serve_forever(self, timeout=0.5):
         self.shutdown()
         logi('No activity in ' + str(emrun_options.silence_timeout) + ' seconds. Quitting web server with return code ' + str(emrun_options.timeout_returncode) + '. (--silence_timeout option)')
         page_exit_code = emrun_options.timeout_returncode
-        emrun_options.kill_on_exit = True
+        emrun_options.kill_exit = True
 
       # If the page has been running too long as a whole, kill process.
       time_since_start = now - page_start_time
       if emrun_options.timeout != 0 and time_since_start > emrun_options.timeout:
         self.shutdown()
         logi('Page has not finished in ' + str(emrun_options.timeout) + ' seconds. Quitting web server with return code ' + str(emrun_options.timeout_returncode) + '. (--timeout option)')
-        emrun_options.kill_on_exit = True
+        emrun_options.kill_exit = True
```

Comment:
I think `kill_on_exit` is clearer, since it literally kills on exit? Otherwise it sounds like "kill the exit" to me...

Reply:
I'm just matching the variable with the command line option name.  There is value  in that alone I think.   Maybe I'll make `--kill_on_exit` and alias for this option.

Reply:
Oh, I missed that there's an external interface here. lgtm either way then.